app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Admin Applications</h1>
+<h3> <%= @application.name %>'s Application</h3>
+
+<section class="pets">
+  <% @application.pets.each do |pet|%>
+    <div id="pet-<%= pet.id %>">
+      <li><%= pet.name %></li>
+      <% if !@application_pet || @application_pet.pet_id != pet.id %>
+        <%= button_to "Approve Application", "/admin/applications/#{@application.id}?pet_id=#{pet.id}&approve=true", method: :patch, local: true%>
+        <%= button_to "Reject Application", "/admin/applications/#{@application.id}?pet_id=#{pet.id}&reject=true", method: :patch, local: true%>
+      <% end %>
```

Comment:
These routes aren't restful if we're thinking about what operation we are performing. We intend to update the status on the join table so the routes should look something like this: `"/admin/applications/#{@application.id}/pets/#{pet.id}?your_key=your_value"`